src/content/api/compilation-hooks.md

```diff
@@ -707,3 +707,52 @@ This hooks allows changing the references reported by dependencies.
 - Callback Parameters: `depRef` `dependency` `module`
 
 W> The `module` parameter will be removed in v5.0.0
+
+### `statsPreset`
+
+`HookMap`
+
+It's a HookMap called for a preset (if used) with an options object. A plugin which handles a preset should set options in the options object. It should avoid overriding existing options.
```

Comment:
Please don't use the concept in its definition: https://en.wikipedia.org/wiki/Circular_definition